docs/howto/openstack/octavia/lbaas-tcp.md

```diff
@@ -0,0 +1,827 @@
+
+=== "OpenStack CLI"
+    To check the operating status of any of the pool members
+    of your load balancer, type something like this:
+
+    ```bash
+    openstack loadbalancer member list mylb-pool
+    ```
+
+    ```plain
+    +-----------+-----------+------------+---------------------+-----------+---------------+------------------+--------+
+    | id        | name      | project_id | provisioning_status | address   | protocol_port | operating_status | weight |
+    +-----------+-----------+------------+---------------------+-----------+---------------+------------------+--------+
+    | c4351167- | srv-      | dfc7004673 | ACTIVE              | 10.15.25. |         61234 | ERROR            |      1 |
+    | fcfa-     | lbaas-1   | 96428bacba |                     | 105       |               |                  |        |
+    | 48fb-     |           | 4376e72cc3 |                     |           |               |                  |        |
+    | 82ac-     |           | e9         |                     |           |               |                  |        |
+    | 76ddfdc73 |           |            |                     |           |               |                  |        |
+    | ca2       |           |            |                     |           |               |                  |        |
+    | 04a32731- | srv-      | dfc7004673 | ACTIVE              | 10.15.25. |         61234 | ONLINE           |      1 |
+    | f3a6-     | lbaas-2   | 96428bacba |                     | 236       |               |                  |        |
+    | 41ca-     |           | 4376e72cc3 |                     |           |               |                  |        |
+    | a638-     |           | e9         |                     |           |               |                  |        |
+    | 6f8767437 |           |            |                     |           |               |                  |        |
+    | 50c       |           |            |                     |           |               |                  |        |
+    +-----------+-----------+------------+---------------------+-----------+---------------+------------------+--------+
+    ```
+
+    In the example above, see the `operating_status` column,
+    specifically the status of server `srv-lbaas-1`. Regarding
+    the operating status of all pool members, you can always
+    limit the scope of your query like so:
+
+    ```bash
+    openstack loadbalancer member list mylb-pool \
+        -c name -c operating_status
+    ```
+
+    ```plain
+    +-------------+------------------+
+    | name        | operating_status |
+    +-------------+------------------+
+    | srv-lbaas-1 | ERROR            |
+    | srv-lbaas-2 | ONLINE           |
+    +-------------+------------------+
+    ```
+
+Finally, let us check what happens when we try to access the
+remote service via the load balancer:
+
+```bash
+wget -q 198.51.100.129:61234 -O -
+```
+
+```plain
+Yello from srv-lbaas-2!
+```
+
```

Comment:
I think you can drive the point home even better if you instruct the learner to run the command three times, like so: (note that the GitHub web interface chops off the closing code fence here; you'll have to re-instate it.)

```suggestion
Finally, let us check what happens when we try to access the
remote service via the load balancer, multiple times:

```console
$ wget -q 198.51.100.129:61234 -O -
Yello from srv-lbaas-2!

$ wget -q 198.51.100.129:61234 -O -
Yello from srv-lbaas-2!

$ wget -q 198.51.100.129:61234 -O -
Yello from srv-lbaas-2!
```

```

Reply:
Sure, I made some changes taking into account your suggestion!